packages/shared/src/modules/tasks/libs/types/task-dto.type.ts

```diff
@@ -1,4 +1,4 @@
-import { type ValueOf } from "../../../../libs/types/types.js";
+import { type ValueOf } from "../../../../libs/types/value-of.type.js";
```

Comment:
Why was this changed?

Reply:
I think I changed this by resolving conflicts using the auto feature in my IDE. I will avoid this.